vscode-wpilib/resources/webviews/gradle2025import.html

```diff
@@ -2,111 +2,186 @@
+          <div class="project-field-container">
+            <input id="projectName" type="text" class="vscode-textfield" />
+            <div id="projectNameError" class="project-error-text">Project name is required</div>
+          </div>
+        </div>
+
+        <div class="project-row">
+          <div class="project-label"><b>Team Number</b></div>
+          <div class="project-field-container">
+            <input
+              id="teamNumber"
+              type="text"
+              inputmode="numeric"
+              class="vscode-textfield"
+              placeholder="Optional"
+            />
+            <div id="teamNumberError" class="project-error-text">Invalid Team Number</div>
+          </div>
+        </div>
+
+        <div class="project-row">
+          <div class="vscode-checkbox">
+            <input id="newFolderCB" type="checkbox" checked />
+            <label for="newFolderCB">
+              <span class="icon">
+                <i class="codicon codicon-check icon-checked"></i>
+              </span>
+              <span class="text">Create a new folder</span>
+            </label>
+            <span class="checkbox-help">Creates a new folder at Base Folder/Project Name</span>
+          </div>
+        </div>
+
+        <div class="project-row">
+          <div class="vscode-checkbox">
+            <input id="desktopCB" type="checkbox" />
+            <label for="desktopCB">
+              <span class="icon">
+                <i class="codicon codicon-check icon-checked"></i>
+              </span>
+              <span class="text">Enable Desktop Support</span>
+            </label>
+            <span class="checkbox-help">Enables desktop simulation and unit testing</span>
+          </div>
+        </div>
+
+        <div class="project-row">
+          <h3>Hardware Option</h3>
+          <p>Select a hardware option if applicable:</p>
+          <div class="radio-option">
+            <input id="romiCB" type="radio" name="hardware" value="romi" />
+            <label for="romiCB">Romi</label>
+          </div>
+          <div class="radio-option">
+            <input id="xrpCB" type="radio" name="hardware" value="xrp" />
+            <label for="xrpCB">XRP</label>
+          </div>
+          <div class="radio-option">
+            <input id="noneCB" type="radio" name="hardware" value="none" checked />
+            <label for="noneCB">SystemCore</label>
+          </div>
+        </div>
+
+        <div class="wizard-navigation">
+          <button id="back-to-step-1" class="vscode-button secondary">Back</button>
+          <button id="next-to-step-3" class="vscode-button next-button">Next</button>
+        </div>
+      </div>
+
+      <!-- Step 3: Review & Import -->
+      <div class="wizard-step" id="step-3">
+        <div class="step-header">
+          <h2>Step 3: Review & Import</h2>
+          <p>Review your settings and select additional options before importing.</p>
```

Comment:
What other options can be selected? I don't see any

Reply:
That refers to the source, destination, and team number. I'm open to alternate wording

Reply:
Shouldn't it just be review your settings before importing?